src/sentry/integrations/jira/webhooks/issue_updated.py

```diff
@@ -13,6 +15,22 @@
 
 
 class JiraIssueUpdatedWebhook(JiraEndpointBase):
+    def handle_exception(self, request: Request, exc: Exception) -> Response:
+        if isinstance(exc, ApiError):
+            if exc.code in (
```

Comment:
@mgaeta should we move this same logic into a mixin?

Reply:
I was thinking that or some util, since it's only useful for Jira